app/app-e2e-tests/e2e/example.spec.ts

```diff
@@ -0,0 +1,135 @@
+import { expect, test } from '@playwright/test';
+import { spawn } from 'child_process';
+const { _electron: electron } = require('playwright');
+
+// const app = await electron.launch({ args: ['main.js'] });
+
+// app.process().stdout.on('data', (data) => console.log(`stdout: ${data}`));
+// app.process().stderr.on('data', (error) => console.log`stderr: ${error}`));
+
+// const electronApp = await electron.launch({ cwd: '/home/vtaylor/headlamp/app/electron', timeout: 0, args: ['main.js'] });
+// electronApp.process().stdout.on('data', (data) => console.log(`stdout: ${data}`));
+// electronApp.process().stderr.on('data', (error) => console.log(`stderr: ${error}`));
+
+// test('launch app', async () => {
+//   const electronApp = await _electron.launch({ cwd: '/home/vtaylor/headlamp/app/electron', args: ['main.js'] })
+//   // close app
+//   await electronApp.close()
+// })
+
+// Function to run a shell command and return a promise
+async function runCommand(command: string, args: string[] = [], options = {}): Promise<void> {
+  return new Promise((resolve, reject) => {
+    const process = spawn(command, args, { ...options, detached: true, stdio: 'ignore' });
+
+    process.on('error', error => {
+      console.error(`Error executing command: ${command}\n${error}`);
+      reject(error);
+    });
+
+    process.on('close', code => {
+      if (code !== 0) {
+        console.error(`Command failed with exit code ${code}`);
+        reject(new Error(`Command failed with exit code ${code}`));
+      } else {
+        resolve();
+      }
+    });
+
+    // Detach the process and resolve immediately
+    process.unref();
+    resolve();
+  });
+}
+
+test('launch app', async () => {
+  // Launch Electron app.
+  const electronApp = await electron.launch({
+    cwd: '/home/vtaylor/headlamp/app/electron',
```

Comment:
We will need a better way to use the absolute path as we cannot just run `headlamp/app/electron` we could also maybe use path?

Reply:
Added changes to solve this